common/src/main/kotlin/feature/actionMenu/AddActionMenu.kt

```diff
@@ -0,0 +1,34 @@
+package feature.actionMenu
+
+import PaginationList
+import action.ActionEnum
+import menu.Backable
+import menu.MenuItem
+import menu.PaginationMenu
+import net.minestom.server.entity.Player
+
+class AddActionMenu(
+    private val backMenu: ActionsMenu
+): PaginationMenu("Add Action"), Backable {
+    override fun paginationList(player: Player): PaginationList<MenuItem> {
+        val menuItems = ArrayList<MenuItem>()
+        for (action in ActionEnum.entries) {
+            // why is this the only way i can find to create the item
+            val item = action.clazz.getDeclaredConstructor().newInstance().createAddDisplayItem()
+            item.action = {
+                backMenu.actions.add(action.clazz.getDeclaredConstructor().newInstance())
+                backMenu.open(player)
+            }
```

Comment:
val action = action.clazz.getDeclaredConstructor().newInstance()
val item = action.createAddDisplayItem()
item.action = {
backMenu.actions.add(action)
backMenu.open(player)
}

Reply:
Makes it so it doesnt create two instances of a class